src/main/java/at/hannibal2/skyhanni/data/MaxwellPowers.kt

```diff
@@ -0,0 +1,41 @@
+package at.hannibal2.skyhanni.data
+
+enum class MaxwellPowers(val power: String) {
```

Comment:
Is there a benefit to doing this as an enum rather than something that could be implemented through the Repo, allowing for quicker responses to Hypixel changes?

Reply:
I'd probably be better if it was a repo thing, buttt I dont wanna do it regex is alr killing mee

Reply:
maxwell powers needs to be a repo.